ts/features/itwallet/common/store/reducers/index.ts

```diff
@@ -100,7 +100,11 @@ export const migrations: MigrationManifest = {
 
   // Removed reissuanceFeedbackBannerHidden from preferences
   "7": (state: PersistedState): PersistedState =>
-    _.omit(state, "preferences.reissuanceFeedbackBannerHidden")
+    _.omit(state, "preferences.reissuanceFeedbackBannerHidden"),
+
+  // Added isBottomSheetForQuitReissuingSurveyVisible to preferences store and set it to true by default
+  "8": (state: PersistedState): PersistedState =>
+    _.omit(state, "preferences.isBottomSheetForQuitReissuingSurveyVisible")
```

Comment:
This migration isn’t correct: `omit` removes attributes from objects, while in this case `set` is the appropriate method.
Also, following the previous comment, we should skip this migration entirely since the new persisted preferences should be optional. 
What do you think?